config/Procfile

```diff
@@ -0,0 +1 @@
+worker: bundle exec sidekiq -C ./config/sidekiq.yml
```

Comment:
You're missing the ending newline in this file.

Reply:
@MahmudH see if this helps https://stackoverflow.com/questions/12415147/ensure-newline-at-eof-in-rubymine

Reply:
👍